### PR TITLE
Fix css alert animation

### PR DESCRIPTION
**Pull request by mvn-loantruong-dn**

### CODE REVIEW

File: src/assets/stylesheet/pages/_home.scss [hunk 1]
```diff
@@ -42,6 +42,16 @@
     }
   }
 }
+.btn-delete {
+  display: inline-block;
+  position: relative;
+  margin-left: 10px;
+  .show-popover {
```
Comment (at-vinguyen) [inline src/assets/stylesheet/pages/_home.scss]: Rank B
It should be contained in popover

**Review by at-vinguyen — CHANGES_REQUESTED** [review]

File: src/assets/stylesheet/modules/_alert.scss [hunk 2]
```diff
@@ -41,3 +41,6 @@
 .alert-danger {
   @include alert-variant(alert-danger-bg, alert-danger-border, white);
 }
+.alert-undefined.fade-out {
```
Comment (at-vinguyen) [inline src/assets/stylesheet/modules/_alert.scss]: Rank B
What is this?

Comment (at-vinguyen) [reply src/assets/stylesheet/modules/_alert.scss]: Scss syntax

File: src/assets/stylesheet/modules/_popover.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    padding: 8px 15px;
+    border-top: 1px solid map-get($color, white-e);
+    text-align: right;
+    .btn-sm {
+      padding: 4px 8px;
+      text-transform: capitalize;
+    }
+  }
+}
+.show-popover {
```
Comment (at-vinguyen) [inline src/assets/stylesheet/modules/_popover.scss]: Rank B
The class modifier should be inside `popover`

File: src/assets/stylesheet/pages/_home.scss [hunk 4]
```diff
@@ -42,6 +42,11 @@
     }
   }
 }
+.btn-delete {
```
Comment (at-vinguyen) [inline src/assets/stylesheet/pages/_home.scss]: Rank B
Naming issue.

File: src/assets/stylesheet/modules/_popover.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    border-color: map-get($color, white) transparent;
+  }
+  .popover-title {
+    background-color: map-get($color, primary);
+    border-bottom: 1px solid map-get($color, white-e);
+    color: map-get($color, white);
+    padding: 8px 15px;
+    line-height: 24px;
+    font-size: 18px;
+    border-top-left-radius: 3px;
```
Comment (at-vinguyen) [inline src/assets/stylesheet/modules/_popover.scss]: Rank B
Using mixin

File: src/assets/stylesheet/modules/_buttons.scss [hunk 6]
```diff
@@ -12,6 +12,7 @@ button, .btn {
   cursor: pointer;
   text-transform: uppercase;
   zoom: 1;
+  line-height: 20px;
```
Comment (at-vinguyen) [inline src/assets/stylesheet/modules/_buttons.scss:15]: Rank C
Why did you adding line-height?

Comment (mvn-loantruong-dn) [reply src/assets/stylesheet/modules/_buttons.scss:15]: Fix lỗi cho bên IE vì 1 button còn 1 cái span nên nó ko đều nhau

**Review by at-vinguyen — APPROVED** [review]

**Pull request merged**